frontend/pulse3d/src/pages/users-info.js

```diff
@@ -322,6 +356,15 @@ export default function UserInfo() {
           />
         </Container>
       </PageContainer>
+      <ModalWidget
+        open={modalState === "generic"}
+        labels={modalLabels.messages}
+        buttons={modalButtons}
+        closeModal={() => {
+          setModalState("");
```

Comment:
You might also need to reset the deactivate modal messages because you're pushing usernames into the array